tests/prefetch.spec.ts

```diff
@@ -51,11 +51,11 @@ test('can prefetch using link props', async ({ page }) => {
 
   await page.getByRole('link', { name: 'On Mount' }).click()
   await isPrefetchPage(page, 2)
-  await expect(requests.requests.length).toBe(0)
+  expect(requests.requests.length).toBe(0)
```

Comment:
`await`s here were not necessary, `expect().toBe()` doesn't return a promise, it's all synchronous